app/src/main/java/org/projectbuendia/client/models/Chart.java

```diff
@@ -16,11 +16,16 @@
 /** A chart definition. */
 public class Chart {
     public final String uuid;  // UUID of the OpenMRS Form containing this chart definition
-    public final List<ChartSection> tileGroups;
-    public final List<ChartSection> rowGroups;
+    public List<ChartSection> tileGroups;
```

Comment:
Ok, now that I've actually read the code, I'd prefer if you kept these as `final`, and made `name` `final` and set as a constructor parameter. Keeping them `final` means that this class is immutable, which makes code significantly easier to reason about - you know that once a class is constructed, nothing is modifying values.

(Ideally, the `List`s would be immutable too, but one step at a time :smile:)